code/controllers/subsystems/records.dm

```diff
@@ -215,6 +215,21 @@
 		if(!department && !(name in manifest["heads"]))
 			manifest["misc"][++manifest["misc"].len] = list("name" = name, "rank" = rank, "active" = isactive)
 
+	for(var/mob/living/silicon/S in player_list)
```

Comment:
This might include maintenance drones.

Reply:
It will, plus mining drones, construction drones. But I think if is okay to have them there? Also along as when they die, they get off the manifest. Currently there is no way to check for list of drones unless you look at miant drone console and it only shows maint drones